modyn/storage/include/internal/file_wrapper/binary_file_wrapper.hpp

```diff
@@ -2,6 +2,7 @@
 
 #include <spdlog/spdlog.h>
 
+#include <cctype>
```

Comment:
For which function is this needed?

Reply:
Does not seem to be needed anymore, thanks!